src/logic/documents/scorecards.js

```diff
@@ -384,7 +384,14 @@ const scorecard = ({
             'ID',
             [
               { text: 'Name', alignment: 'left', width: 'auto' },
-              { text: competitor.wcaId || ' ', alignment: 'right' },
+              {
+                text:
+                  competitor.wcaId ||
+                  // If the competitor has a name, then this is a new competitor
+                  // Else this is a blank scorecard
+                  (competitor.name.trim().length > 0 ? 'New Competitor' : ' '),
```

Comment:
```suggestion
                  (competitor.name.trim().length > 0 ? 'New competitor' : ' '),
```

Reply:
Btw there are two places where we have `name: ' '`, could you change name and registrantId to `null` and then do `|| ' '` where we use them? This will also simplify this ternary :)

Reply:
Of course - sounds good!